sources/packages/api/src-sql/EducationProgramsOfferings/Alter-education-program-offerings.sql

```diff
@@ -0,0 +1,73 @@
+-- Add year_of_study to education_programs_offerings table.
+ALTER TABLE
+    sims.education_programs_offerings
+ADD
+    COLUMN IF NOT EXISTS year_of_study INT NOT NULL DEFAULT 1;
+
+COMMENT ON COLUMN sims.education_programs_offerings.year_of_study IS 'Year of study of the study period.';
+
+-- Add show_yr_of_study to education_programs_offerings table.
+ALTER TABLE
+    sims.education_programs_offerings
+ADD
+    COLUMN IF NOT EXISTS show_yr_of_study BOOLEAN;
```

Comment:
Can it be renamed to _show_year_of_study_?

Reply:
Renaming of db column, entity, DTO and form.io done